client/lclusterc.go

```diff
@@ -0,0 +1,316 @@
+/*
+ * File: client/main.go
+ *
+ * Description: Contains the lcluster client code.
+ *
+ * TODO: removejob is broken, others funcs are unimplemented
+ */
+
+package main
```

Comment:
Adjusted comments of funcs and types and definitions to be full sentences.

https://github.com/rbisewski/lclusterd/pull/2/commits/c90fc34eb3263f54028ba3b67c572b4a2bc2e3e1